cfme/tests/containers/test_properties.py

```diff
@@ -132,8 +134,7 @@
 ]
 
 
-@pytest.mark.parametrize('test_item', TEST_ITEMS,
-                         ids=[ti.args[1].pretty_id() for ti in TEST_ITEMS])
+@pytest.mark.parametrize('test_item', TEST_ITEMS)
```

Comment:
causes problem when there is a bugzilla marker, it's too complicating the way to call that (sometimes args[1].pretty_id, sometimes args[0].args[1].pretty_id). I can create some function that treat this but i think it's too much for this purpose. what you think?